Code/Tools/SceneAPI/SceneData/Rules/TagRule.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
```

Comment:
Copyright header should be at the top of the file, else the automated check will fail